util/util.go

```diff
@@ -261,6 +261,20 @@ func TimestampWithinLimit(latest time.Time, ts string, limit time.Duration) bool
 	return deadline.After(latest)
 }
 
+func TimestampAfterTimestamp(after string, before string) bool {
```

Comment:
I just think backwards, I suppose, but I would have intuitively ordered the arguments as (before, after).

Reply:
Not sure what the "right" approach is here. I tried both at one point or another.

I read it as `TimestampAfterTimestamp` should answer the question, "Is the first TIMESTAMP I provide AFTER the second TIMESTAMP I provide?" Maybe it adds the the confusion that I name the variables `after` and `before` (presupposing that the first time will be after the second time until proven otherwise). I will change the names to `time1` and `time2` in the next commit.